.github/workflows/node.js.yml

```diff
@@ -2,7 +2,7 @@ name: Node.js CI
 
 on:
   pull_request:
-    branches: release
+    branches: [ master, release ]
```

Comment:
I think it would be more elegant to do `publish` on `push` to the release branch. And `make` could actually be done for all branches, both push and pull request.